aspnetcore/mvc/views/razor.md

```diff
@@ -1004,6 +1004,20 @@ Developers are encouraged to match the casing of file and directory names to the
 
 Matching case ensures the deployments find their views regardless of the underlying file system.
 
+## Imports used by Razor
+
+The following imports are generated by the ASP.NET Core web templates to support Razor Files:
+
+```
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.AspNetCore.Mvc.ViewFeatures;
+```
```

Comment:
Looks good! @captainsafia, I noticed that some namespaces are automatically imported into Razor files in ASP.NET Core projects but I don't think it was documented anywhere previously. Can you confirm this is the right set of namespaces to document?